server/auth_test.go

```diff
@@ -42,7 +42,7 @@ var configAuth2 = server.Configuration{
 func TestMissingAuthToken(t *testing.T) {
 	helpers.AssertAPIRequest(t, nil, &configAuth, &helpers.APIRequest{
 		Method:   http.MethodGet,
-		Endpoint: server.OrganizationsEndpoint,
+		Endpoint: configAuth.APIPrefix + "organizations/1/clusters",
```

Comment:
it's better to set api prefix in config, it's gonna be added to url by AssertAPIRequest automatically

Reply:
I don't quite understand what do you mean?

Reply:
we're already adding apiprefix from server config here https://github.com/RedHatInsights/insights-results-aggregator/blob/master/tests/helpers/http.go#L82

Reply:
ok, so I'll remove it